opengrok-web/src/main/webapp/js/utils-0.0.27.js

```diff
@@ -1117,7 +1117,7 @@
                             $ul.append(
                                     $('<li>')
                                     .addClass('message-group-item')
-                                    .addClass(tag.messages[j].class)
+                                    .addClass(tag.messages[j].cssclass)
```

Comment:
I'm little bit lost now with the renaming, but isn't this currently "cssClass" again?